mapiproxy/servers/default/emsmdb/emsmdbp_object.c

```diff
@@ -4,6 +4,7 @@
    EMSMDBP: EMSMDB Provider implementation
 
    Copyright (C) Julien Kerihuel 2009-2014
+   Copyright (C) Enrique J. Hernández 2015
```

Comment:
Add the Copyright change to the other PR instead.

I want us to remain consistent with the way we add or not a new contributor to the file. Your other contribution definitely  requires this change. This one doesn't.